### ISSUE TITLE
Adicionar Contato de Atendimento em Lista de Campanhas

### ISSUE THREAD

**Issue by hildors** [ticketz-oss/ticketz#157]
Acredito que seria muito produtivo poder adicionar um contato de atendimento (menu Contatos) diretamente para uma Lista das Campanhas. Se isso fosse direto na listagem de Contatos seria ótimo, porém tendo mais de uma lista de campanhas poderia ter a necessidade de um select + botão de ação. 
Fica a sugestão de recurso.

**Comment by allgood** [comment]
Não entendi bem. Seria incluir em uma lista de campanha um contato presente na lista de atendimentos?

**Comment by hildors** [comment]
Sim.  Você estando na página de Contatos, ter um botão em cada usuários para adiciona-lo em uma lista de campanhas.
